src/cat-life-event.tsx

```diff
@@ -6,9 +6,40 @@ export type Props = {
 
 export default (): FC<Props> => {
   const component = (props: Props): ReactElement => {
-    return <>
-      {props.children}
-    </>
+    return <div>
```

Comment:
`div` works for a lot of things, but I think there's a more semantic tag for this. Have you looked into `section`?

Here's MDN's docs on it: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section

Specifically I'm looking at this:

> The <section> [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) element represents a generic standalone section of a document, which doesn't have a more specific semantic element to represent it.

```suggestion
    return <section>
```

But also you'll need to cover the end tag if you apply this change.